engine/src/main/java/org/terasology/config/flexible/FlexibleConfigManager.java

```diff
@@ -17,14 +17,46 @@
 
 import org.terasology.engine.SimpleUri;
 
+/**
+ * Stores multiple {@link FlexibleConfig} instances that can be retrieved using their id.
+ * Also responsible for overseeing their serialization - to and from - disk.
+ */
 public interface FlexibleConfigManager {
+    /**
+     * Adds the given {@link FlexibleConfig} to this manager.
+     * This method throws a RuntimeException if a FlexibleConfig with the given id already exists.
```

Comment:
I think you can move the content of this line and put it besides the @throws line below, isn't it?